lib/media/drm_engine.js

```diff
@@ -1638,24 +1639,28 @@ shaka.media.DrmEngine = class {
       return;
     }
     shaka.log.debug('Unwrapping PlayReady request.');
-    const dom = shaka.util.XmlUtils.parseXmlString(xml, 'PlayReadyKeyMessage');
+    const dom = TXml.parseXmlString(xml, 'PlayReadyKeyMessage');
     goog.asserts.assert(dom, 'Failed to parse PlayReady XML!');
 
     // Set request headers.
-    const headers = dom.getElementsByTagName('HttpHeader');
+    const headers = TXml.getElementsByTagName(dom, 'HttpHeader');
     for (const header of headers) {
-      const name = header.getElementsByTagName('name')[0];
-      const value = header.getElementsByTagName('value')[0];
+      const name = TXml.getElementsByTagName(header, 'name')[0];
+      const value = TXml.getElementsByTagName(header, 'value')[0];
       goog.asserts.assert(name && value, 'Malformed PlayReady headers!');
-      request.headers[name.textContent] = value.textContent;
+      request.headers[
+          /** @type{string} */(shaka.util.TXml.getTextContents(name))] =
+        /** @type{string} */(shaka.util.TXml.getTextContents(value));
```

Comment:
nit: add two spaces in the indentation

Reply:
The linter will autofix it to look like this no matter what spacing I put.

Reply:
ok!